AssistantBOT.py

```diff
@@ -4296,8 +4295,7 @@ def messaging_parse_flair_response(subreddit_name, response_text, post_id):
         # Returns as tuple `('FLAIR' (text), INT)`
         # If the match is higher than or equal to `min_fuzz_ratio`, then
         # assign that to `returned_template`. Otherwise, `None`.
-        best_match = process.extractOne(response_text, list(lowercased_flair_dict.keys()),
-                                        scorer=fuzz.WRatio)
```

Comment:
the scorer argument is not supported in rapidfuzz yet (will be in one of the next releases), but WRatio is the default scorer anyways

Reply:
Thanks for the PR! And for writing RapidFuzz! I will merge this back in once I get back to my home in a week. :)